src/scheduler/scheduler.ts

```diff
@@ -113,7 +113,7 @@ class DetachableLoop {
 
     async startLoop(): Promise<void> {
         // See if the exec time is available in durable storage...
-        if (this.schedMode !== SchedulerMode.ExactlyOncePerInterval)
+        if (this.schedMode == SchedulerMode.ExactlyOncePerInterval)
```

Comment:
Triple equals?

Reply:
Python equals.

Reply:
We must not be using the eqeqeq rule?

Reply:
25 existing occurrences... another PR I guess.